net.nmlgc.rec98.sh01.yaml

```diff
@@ -0,0 +1,112 @@
+id: net.nmlgc.rec98.sh01
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm19
+build-options:
+  append-path: /usr/lib/sdk/llvm19/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm19/lib
+finish-args:
+  - --device=dri
+  - --device=input
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+cleanup:
+  - /bin/pango*
+  - /bin/tup*
+  - /include
+  - /lib/*.a
+  - /lib/libc++.modules.json
+  - /lib/pkgconfig
+  - /share/libc++
+  - /share/man
+
+command: GIAN07
+modules:
+  # See https://aur.archlinux.org/packages/ttf-ipa-mona
+  - name: ttf-ipa-mona
+    sources:
+      - type: archive
+        url: https://web.archive.org/web/20111108024313if_/http://www.geocities.jp/ipa_mona/opfc-ModuleHP-1.1.1_withIPAMonaFonts-1.0.8.tar.gz
+        dest-filename: ttf-ipa-mona.tar # It's actually uncompressed!
+        sha256: cb7b6975faf5caa070ffa1699a241d1c093f3b74c54696e5af23b5bc3b118a44
+    buildsystem: simple
+    subdir: fonts
+    build-commands:
+      - install -Dm644 ipag-mona.ttf -t "$FLATPAK_DEST/share/fonts/TTF/"
+      - install -Dm644 COPYING.font.ja -t "$FLATPAK_DEST/share/licenses/ttf-ipa-mona/"
+
+  - name: pango
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/pango/-/archive/1.56.0/pango-1.56.0.tar.bz2
+        sha256: d444ef9a8cec24b6ffc989a4e23bc9c60c290b7218327847a4e712e1c7a9ca70
+      - type: patch
+        path: pango-workaround-harfbuzz-issue-4311.patch
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-testsuite=false
+      - -Dbuild-examples=false
+      - -Dintrospection=disabled
+      - -Dxft=disabled
+
+  # Required for C++23 Standard Library Module support, and not part of the
+  # LLVM SDK extension.
+  - name: libcxx
+    buildsystem: cmake-ninja
+    builddir: true
+    subdir: runtimes
+    sources:
+      - type: archive
+        url: https://github.com/llvm/llvm-project/releases/download/llvmorg-19.1.6/llvm-project-19.1.6.src.tar.xz
+        sha256: e3f79317adaa9196d2cfffe1c869d7c100b7540832bc44fe0d3f44a12861fa34
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_C_COMPILER=clang
+      - -DCMAKE_CXX_COMPILER=clang++
+      - -DLLVM_ENABLE_RUNTIMES=libcxx;libcxxabi;libunwind
+
+  - name: tup
```

Comment:
missing the cleanup rule to remove it all as it is build only.

Reply:
I don't see a cleanup here, add

```yaml
cleanup:
- "*"
```

to the module here, it will remove all the contents of this module post manifest build.